include/odp/api/abi-default/spinlock.h

```diff
@@ -19,7 +19,7 @@ extern "C" {
 
 /** @internal */
 typedef struct odp_spinlock_s {
-	char lock;  /**< lock flag, should match odp_atomic_flag_t */
+	int lock;  /**< lock flag, should match odp_atomic_flag_t */
```

Comment:
Do you plan to reimplement odp-dpdk's spinlocks using `rte_spinlock`? Any reason for doing that?
Another question regarding spinlocks. Right now we have plain spinlocks and ticketlocks, which serve exactly the same purpose. Maybe we should drop one of the interfaces/implementations and have just one spinning lock implementation?